### DISCUSSION TITLE
How to update workflows?

### DISCUSSION THREAD

**Discussion by tdari** [Tauffer-Consulting/domino#218] [Q&A] [answered]
Is there any option to update workflows. I recently started to use the product please let me know if there is any option.
Thanks.

**Comment by vinicvaz** [comment, marked as answer]
Hey @tdari .
Unfortunately we don't support workflows update now, but it is on our features list to be implemented ASAP. 
I truly believe this feature is a good advance in the workflows build/test life cycle and it is important to have it soon. It is not available yet because the current architecture makes it a bit tricky to implement so it will be a major change, probably it will come in the next major release. 
We are focusing our efforts on that since almost all the users from Domino have been requested this feature!
Once I have a PR you can track I paste it in this discussion.

Thanks!

**Reply by tdari** [reply]
Thanks @vinicvaz, I look forward to see next updates. I would like to contribute to the product if you are accepting contributions. But I can't see who is working on what. Let me know if you are planning to use the project section on Github.

**Reply by vinicvaz** [reply]
@tdari Sure! We accept any kind of contributions, you are totally welcome!
I just opened this more detailed issue to this feature https://github.com/Tauffer-Consulting/domino/issues/219 it will help us to discuss the best way to that.
We are using the project section on Github but I think it is private, not sure, I'll check it, until there we can track the progress in the issue.
